packages/shared/src/common/date.ts

```diff
@@ -15,3 +15,10 @@ export const optionalDateSchema = z
   .trim()
   .nullish()
   .refine(isValidISODateOptional, { message: "Invalid ISO date" });
+
+export const diffFromNow = (date1?: Date, format: dayjs.UnitTypeLong = "millisecond"): number => {
```

Comment:
2 nits: 
- i think something like `calculateElapsedTime` would be a better name, and
- why `date1` and not just `date`?

Reply:
I like `diffFromNow` b/c its short and good enough to provide context to what it does - considering we're passing a date time as parameter that one can infer we're talking about time.

I also like the more expressive suggestion @RamilGaripov did, but then I'd name it `elapsedTimeFromNow` since we only pass one param... (no need for "calculate" IMO).